frontend/src/metabase/env.ts

```diff
@@ -1,15 +1,45 @@
+const tryOrDefault = <T>(fn: () => T, defaultValue: T): T => {
+  try {
+    return fn();
+  } catch (e) {
```

Comment:
I'm not 100% sure how the property accesses could throw an error, is this because `process.env` could be undefined?

Reply:
I see! It makes sense that the tests doesn't have the env defined as I imagine most of those envs applied to the built bundle, and not the tests itself.